sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1094,21 +1099,43 @@ def delete_blobs(self, *blobs, **kwargs):
             be supplied, where each value is either the name of the blob (str) or BlobProperties.
 
             ..note::
-                When then blob type is dict, here's a list of keys you can set:
-                blob name: 'name'
-                container name: 'container'
-                snapshot you want to delete: 'snapshot'
-                whether to delete snapthots when deleting blob: 'delete_snapshots'
-                if the blob modified or not: 'if_modified_since', 'if_unmodified_since'
-                match the etag or not: 'etag', 'match_condition'
-                lease id or lease client: 'lease_id'
-                timeout for subrequest: 'timeout'
+                When the blob type is dict, here's a list of keys, value rules:
+                blob name:
+                    key: 'name', value type: str
+                container name:
+                    key: 'container', value type: str
```

Comment:
I'm not sure we should support `container` here. At least not initially. This kind've breaks the concept of running this command from a specific container client. What are the other languages supporting here?

Reply:
Though it doesn't make much sense because it's at container level, I guess it's less redundant and confusing than adding a new one at service level?

Reply:
We can leave this one here to only support deleting blobs in the same container, and add a new api which supports delete blobs in different containers at service level, let me know how you feel about it!